### ISSUE TITLE
Spaces between closing parens

### ISSUE THREAD

**Issue by copy** [ocaml-ppx/ocamlformat#517]
Currently ocamlformat puts closing parens on the same line (Lisp style). It does, however, put spaces between the closing parens. I'm not a Lisp developer, but this style looks odd to me.

For example:

```ocaml
;;
bar (fun () ->
    if foo then (
      barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar () ;
      barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar () ) )
```

**Comment by jberdine** [comment]
It seems that `Conf.indicate_multiline_delimiters` could be tested in a few more place.

**Comment by copy** [comment]
@gpetiot Yes, that's exactly what I was looking for. Cheers!

**Comment by gpetiot** [comment]
Closed with #526
